mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowContext.java

```diff
@@ -31,6 +31,7 @@
 public class MlflowContext {
   private MlflowClient client;
   private String experimentId;
+  public static String defaultRepoNotebookExperimentId;
```

Comment:
Does this string need to exist? If so, does it need to be public?

Reply:
It's translated from python where we cache the default repo experiment for a single python session to avoid having to make an API call every time. I'm ok removing it for java if you'd like

Reply:
Doesn't need to be public though

Reply:
I see. Makes sense to me - can we make it private and add an inline comment?